src/high_level/pwhash.rs

```diff
@@ -844,16 +894,21 @@ mod public {
             salt_mod[0..16].copy_from_slice(&[0u8; 16]);
             let modified = PasswordHash::from_slice(&pwd_mod, &salt_mod, 3, 4096).unwrap();
 
-            assert!(hash_password_verify(&modified, &password, 3, 4096).is_err());
+            assert!(hash_password_verify(&modified, &password).is_err());
```

Comment:
I think it would be good to add a test, similar to the suggestion in the doc example, that shows modifying the `iterations` and `memory` parameters (individually and at the same time), also gives a verification error.

Reply:
I'm not sure how to do this without using the `hazardous` module. Is that what you're suggesting?

Reply:
I was thinking more in the lines of taking one of the existing, valid, encoded password hashes and manually modifying the iterations/memory parameters therein (modifying within in the string). Like one of the encoded strings you've removed here as a starting point perhaps?

Reply:
Modifying the encoded parameters turned out to be a good way to do it. Nice and simple. Good idea